resources.yaml

```diff
@@ -80,7 +80,7 @@ deployment:
     image: htcondor-secondary
     secondary_htcondor_cluster: true
   worker-c28m225-htcondor-secondary:
```

Comment:
This flavor is used only for the training. We do not use them for our regular queue.